GOVERNANCE.md

```diff
@@ -0,0 +1,176 @@
+
+## On / Offboarding
+
+The On / Offboarding section is informational and can be changed by [lazy consensus](#consensus) unless challenged. If no consensus can be reached, the matter may be resolved by [majority](#majority-vote) vote.
+
+### Onboarding
+
+Once a new member is approved they must enable 2 factor authentication for their Github account prior to starting the onboarding process. Then, the new member is:
+
+* Added to the list of team members, ideally by sending a PR of their own, or at least approving said PR
+* Announced in the *#metrictank-dev* channel in [slack][slack] and the Metrictank [development mailing list][devlist] by an existing team member
+* Added to the [Github project][gh] as a collaborator. Optionally, they are added to the community, and related organizations and repositories
+* Added to the private [team mailing list][team]
+
+### Offboarding
+
+The ex-member is:
+
+* Removed from the list of team members. Ideally by sending a PR of their own, at least approving said PR. In case of forced removal, no approval is needed
+* Removed from the [Github project][gh] and related organizations and repositories
+* Removed from the private [team mailing list][team]
+* Not allowed to call themselves an active team member any more, nor allowed to imply this to be the case
+* Added to a list of [alumni][alumni] if they so choose
+* If needed, we reserve the right to publicly announce removal
+
+## FAQ
+
+The FAQ section is informational and can be changed by [lazy consensus](#consensus) unless challenged. If no consensus can be reached, the matter may be resolved by [majority](#majority-vote) vote.
+
+### How do I propose a decision?
+
+Open an issue in Github with your motion. If there are objections and no consensus can be found, a vote may be called by a team member.
+
+### How do I become a team member?
+
+To become an official team member, you should make ongoing contributions to one or more project(s) for at least three months. At that point, a team member may propose you for membership. The discussion about this will be held in private, and you will be informed privately when a decision has been made.
+
+Should the decision be in favor, your new membership will also be announced in the *#metrictank-dev* channel in [slack][slack] and on the Metrictank [development mailing list][devlist].
+
+### How do I remove a team member?
+
+Team members may resign by notifying the private [team mailing list][team]. If you think a team member should be removed against their will, propose this to metrictank.team@gmail.com. Discussions will be held in the private [team mailing list][team].
```

Comment:
I think every occurence of `[team]` should be `[teamlist]`

Reply:
I think that would make it cluttered with links when there are already plenty which link to the team members document.

Reply:
Huh?
1) My suggestion does not change the amount of links. You already have links.
2) `[team]` is a broken link. There is no entry "team" in the list below (only "teamlist") and so your links don't work

Reply:
Ahhh thanks @Dieterbe